src/control/createIndex.ts

```diff
@@ -35,6 +35,7 @@ const CreateIndexOptionsProperties: CreateIndexOptionsType[] = [
   'deletionProtection',
   'waitUntilReady',
   'suppressConflicts',
+  'tags',
```

Comment:
Would it be worth adding any `createIndex` unit test coverage for this? I know you're integration testing as well, so maybe not.

Reply:
Wouldn't hurt